api/alerts.go

```diff
@@ -34,6 +35,20 @@ type Alert struct {
 	Type string `json:"type"`
 }
 
+func (a *Alert) String() string {
+	return fmt.Sprintf(
```

Comment:
Note to self: Maybe it would be a good idea to use the same or similar output as the Plugin provides at the end. 

This way users know what string they have to work with

```
output := fmt.Sprintf("%s [%s] %s (%s) %s",
 alert.RaisedAt.Local().Format("2006-01-02 15:04 MST"),
 alert.Severity, agentName, alert.Product, alert.Description)
```

Reply:
Done